src/NHibernate.Test/TestDialects/PostgreSQL83TestDialect.cs

```diff
@@ -21,13 +16,5 @@ public override bool SupportsNullCharactersInUtfStrings
         {
             get { return false; }
         }
-
-        /// <summary>
-        /// Npgsql's DTC code seems to be somewhat broken as of 2.0.11.
-        /// </summary>
-        public override bool SupportsDistributedTransactions
```

Comment:
But, it is available since long. Tests succeed. It was failing previously not because Postgres was not supporting it, but because we were not at all enlisting Postgres connection into scopes: enlist is `false` by default with npgsql. This PR switch it to `true`.